package.json

```diff
@@ -17,7 +17,7 @@
   },
   "homepage": "https://github.com/conbrad/renovate-test#readme",
```

Comment:
Was it intentional to have the homepage set to your personal Github repo?

Reply:
No, `npm init` must have set it.